src/main/java/com/github/instagram4j/instagram4j/requests/tags/search/SearchTagRequest.java

```diff
@@ -0,0 +1,60 @@
+package com.github.instagram4j.instagram4j.requests.tags.search;
+
+import com.github.instagram4j.instagram4j.IGClient;
+import com.github.instagram4j.instagram4j.requests.IGGetRequest;
+import com.github.instagram4j.instagram4j.responses.tags.search.SearchTagResponse;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import java.util.Optional;
+
+@AllArgsConstructor
+@NoArgsConstructor
+public class SearchTagRequest extends IGGetRequest<SearchTagResponse> {
```

Comment:
This request should fall under the "tags" package (Not the tags.search package)

Additionally this request name must also follow the path name and then request at the end, which is 
TagsSearchRequest

Reply:
Please, I don't understand how it works. 
![Uploading Screenshot_20201128_052959.jpg…]()

Reply:
> Please, I don't understand how it works.

Please do not comment on review comments about this issue. We have a chat for support which you can join for help
[![Discord](https://img.shields.io/discord/777326687326044171?label=chat)](https://discord.com/invite/gR2EPwZGEZ)

Reply:
Fixed